pnpm-lock.yaml

```diff
@@ -1299,6 +1299,9 @@ packages:
   /@lit-labs/ssr-dom-shim@1.0.0:
     resolution: {integrity: sha512-ic93MBXfApIFTrup4a70M/+ddD8xdt2zxxj9sRwHQzhS9ag/syqkD8JPdTXsc1gUy2K8TTirhlCqyTEM/sifNw==}
 
+  /@lit-labs/ssr-dom-shim@1.1.0:
```

Comment:
You may want to use `pnpm dedupe` to remove the 1.0.0 version and align everything on the 1.1.0 one.